src/python/WMCore/MicroService/DataStructs/DefaultStructs.py

```diff
@@ -68,7 +68,8 @@
                           error="",
                           total_num_requests=0,
                           clean_num_requests=0,
-                          archived_num_requests=0,
+                          proper_archived_num_requests=0,
```

Comment:
should we rename it here as well to `normal_archived...`?

Reply:
Agreed